webapp/migrations/Version20240629154640.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20240629154640 extends AbstractMigration
+{
+    private const NEW_ROLES = ['api_problem_editor' => 'API Problem Editor',
+                               'api_contest_editor' => 'API Contest Editor'];
+
+    public function getDescription(): string
+    {
+        return "Add new roles to the database.
+                Problem editor can add/delete/edit anything related to problems; files, testcases.
```

Comment:
I don't think any (end) user will read this and I think @eldering his feedback is that people now don't know what the roles do without checking the code.

Reply:
Exactly. What the roles do should ideally be clear from the role constant names, but at least from the role descriptions .